java/core/src/main/java/com/microsoft/bond/protocol/SimpleBinaryReader.java

```diff
@@ -204,5 +204,11 @@ public void skipWString() throws IOException {
     public InputStream cloneStream() throws IOException {
         return Cloning.cloneStream(this.reader.inputStream);
     }
+
+    @Override
+    public UntaggedProtocolReader cloneProtocolReader() throws IOException {
```

Comment:
Clone is inherited from Object and can't be overridden with this throws.

---
In reply to: [134026347](https://github.com/Microsoft/bond/pull/571#discussion_r134026347) [](ancestors = 134026347)